openapi/gateway.json

```diff
@@ -1087,6 +1170,35 @@
           "version"
         ]
       },
+      "InstallinatorImageId": {
+        "type": "object",
+        "properties": {
+          "control_plane": {
+            "type": "array",
+            "items": {
+              "type": "integer",
+              "format": "uint8",
+              "minimum": 0
+            },
+            "minItems": 32,
+            "maxItems": 32
+          },
+          "host_os": {
+            "type": "array",
+            "items": {
+              "type": "integer",
+              "format": "uint8",
+              "minimum": 0
+            },
+            "minItems": 32,
+            "maxItems": 32
+          }
```

Comment:
I think a hash should probably be hex-encoded, making it easy to copy/paste hashes found in wire dumps. Not sure what we do elsewhere in these cases or how easy it is to achieve this.

Reply:
We have many cases where this isn't the case unfortunately and it's frustrating. But I also really dislike using strings everywhere. See my comment above.